src/Microsoft.Graph.Core/Requests/Middleware/AuthenticationHandler.cs

```diff
@@ -113,7 +114,7 @@ private async Task<HttpResponseMessage> SendRetryAsync(HttpResponseMessage httpR
         /// <returns></returns>
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage httpRequestMessage, CancellationToken cancellationToken)
```

Comment:
this handler is not needed anymore since we have the authentication provider now

Reply:
To avoid making the PR too large and easier to review(and still have it compile) My intention is to refactor all the auth code via #334. 
If it's okay you will see this is done there.